agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -52,13 +56,42 @@ while true; do
 			shift;
 		fi
 		;;
+		--sysinfo)
+		shift;
+		if [ -n "$1" ]; then
+			sysinfo="$1"
+			shift;
+		fi
+		;;
+		--check)
+		shift;
+		if [ -n "$1" ]; then
+			check="$1"
```

Comment:
I don't think we need an argument for `--check`, just its presence on the command line should be sufficient.